acceptance/tests/facts/el.rb

```diff
@@ -58,6 +64,10 @@
       os_arch                 = 'ppc64le'
       os_hardware             = 'ppc64le'
       processor_model_pattern = 'POWER8'
+    elsif agent['platform'] =~ /aarch64/
+      os_arch                 = 'aarch64'
+      os_hardware             = 'aarch64'
+      processor_model_pattern = '' # aarch64 does not populate a model value in /proc/cpuinfo
```

Comment:
Is there any other similar field in `/proc/cpuinfo` that we could try to populate model from?

Reply:
There's not unfortunately. This is all we get:

[root@centos-7-goldimg ~]# cat /proc/cpuinfo 
processor  : 0
BogoMIPS  : 200.00
Features  : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics cpuid
CPU implementer  : 0x43
CPU architecture: 8
CPU variant  : 0x1
CPU part  : 0x0a1
CPU revision  : 1

processor  : 1
BogoMIPS  : 200.00
Features  : fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics cpuid
CPU implementer  : 0x43
CPU architecture: 8
CPU variant  : 0x1
CPU part  : 0x0a1
CPU revision  : 1

Reply:
Dang. Oh well. Maybe something in sysfs?

I'm gonna file a separate bug to dig into getting more processor details on ARM, but for now I'm fine letting this land as-is